src/utils/utils.js

```diff
@@ -15,3 +19,37 @@ export const setNativeValue = (element, value) => {
     valueSetter.call(element, value);
   }
 };
+
+export const getPxeBootPatch = vm => {
+  const patch = [];
+  const annotations = get(vm, 'metadata.annotations', {});
+  if (annotations[ANNOTATION_FIRST_BOOT]) {
+    if (annotations[ANNOTATION_FIRST_BOOT] === 'true') {
+      patch.push({
+        op: 'replace',
+        path: `/metadata/annotations/${ANNOTATION_FIRST_BOOT}`,
+        value: 'false',
+      });
+    } else {
+      // find bootable disk and change boot order
+      const bootableDiskIndex = getDisks(vm).findIndex(disk => disk.bootOrder === 2);
```

Comment:
1 and 2 could be shared constants between utils.test.js

Reply:
created constants out of 1 and 2 and they are now used in utils/utils.test/request